about.html

```diff
@@ -0,0 +1,213 @@
+                    We welcome you to our networking dinner in Rwanda
+
+                </p>
+            </div>
+            <div class="headline-time">
+                <p>Please contact us for further questions about Africam Movies Festival</p>
+            </div>
+            <div class="headline-venue">
+
+                <h5>afrimovies@africa.com</h5>
+            </div>
+        </div>
+    </section>
+    <section class="about-logo">
+        <div class="about-logo-container">
+            <div class="about-logo-title">
+                <h2>African Movies Festival 2023 Logo</h2>
+                <hr class="line">
+            </div>
+            <div class="desk-about-logo-title">
+                <h2>African Movies Festival 2023 Logo</h2>
+                <hr class="line">
+            </div>
+            <div class="about-logo-paragraph">
+                <p>
+                    African Movies Festival introduced a new logo -: an expression of the diversity in our people and
+                    film makers. We believe that Africa is one, and the only language that brings film makers together
+                    is "action." Our mission is to unify African movie makers and build a future of movie industry that
+                    embodies our culture, connection, and community.
+                </p>
+            </div>
+            <div class="about-logo-image">
+                <img src="./images/download.jpg" alt="">
+            </div>
+        </div>
+
+    </section>
+
+    <section class="stories">
+        <div class="stories-container">
+            <div class="story-title">
+                <h2>See our previous events and projects</h2>
+                <hr class="line">
+            </div>
+            <div class="story-paragraph">
+                <p>African Movies endeavors to make Africa the Film making center </p>
+            </div>
+
+
+            <div class="event-area">
+                <div class="event-pic">
+                    <div class="pic-1">
+                        <img src="./images/students.png" alt="image">
+                    </div>
+                    <div class="text">
+                        <h5>2022</h5>
+                        <p>Introduce movies and film in campus</p>
+                    </div>
+                </div>
+
+                <div class="event-pic">
+                    <div class="pic-1">
+                        <img src="./images/conference.png" alt="image">
+                    </div>
+                    <div class="text">
+                        <h5>2022</h5>
+                        <p>Previous Film event</p>
+                    </div>
+                </div>
+            </div>
+
+
+        </div>
+
+
+        <!-- desktop  story images container-->
+
+        </div>
+
+    </section>
+
+    <section class="partners container" id="footer">
+        <div class="partners-content">
+        <h2>Partners</h2>
```

Comment:
- [ ] @kazaneza great work implementing the changes requested however this title color doesn't match the design given, also the opacity:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/212415429-1fed6bcf-321d-4034-9952-bfd5ec4d823d.png) | ![image](https://user-images.githubusercontent.com/61408860/212415486-f77930af-e9a8-427b-ad60-86b81ef3341c.png)